src/app/_components/FeeSection/index.tsx

```diff
@@ -110,7 +123,7 @@ function FeeData({
   );
 }
 
-function getFeeDescription(txType: keyof MempoolFeePriorities) {
+function getFeeDescription(txType: 'all' | 'token_transfer' | 'contract_call' | 'smart_contract') {
```

Comment:
I think we should use the type keyof MempoolFeePriorities here as these hardcoded strings can get out of date